src/features/defi/components/Overview/Overview.tsx

```diff
@@ -226,6 +234,17 @@ export const Overview: React.FC<OverviewProps> = ({
                 </StatGroup>
               )}
               {postChildren}
+              {provider === DefiProvider.ThorchainSavers ? (
```

Comment:
blocking: this will display for RUNEPool too, which isn't currently halted
<img width="522" alt="image" src="https://github.com/user-attachments/assets/198b6782-9068-457f-a409-59c33fa7f558" />

Reply:
Changed in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)